### PR TITLE
Add test cases and examples of HanaOnAzure.

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description

Add test cases and examples of HanaOnAzure

### CODE REVIEW

File: src/HanaOnAzure/docs/New-AzSapMonitorProviderInstance.md [hunk 1]
```diff
@@ -35,23 +35,27 @@ Creates a provider instance for the specified subscription, resource group, SapM
 
 ## EXAMPLES
 
-### Example 1: {{ Add title here }}
+### Example 1: Create an instance of SAP monitor by string for HANA
 ```powershell
-PS C:\> {{ Add code here }}
+PS C:\> New-AzSapMonitorProviderInstance -ResourceGroupName nancyc-hn1 -Name ps-sapmonitorins-t01 -SapMonitorName yemingmonitor -Type SapHana -HanaHostname 'hdb1-0' -HanaDatabaseName 'SYSTEMDB' -HanaDatabaseSqlPort 30015 -HanaDatabaseUsername SYSTEM -HanaDatabasePassword (ConvertTo-SecureString "Manager1" -AsPlainText -Force)
```
Comment (isra-fel) [inline src/HanaOnAzure/docs/New-AzSapMonitorProviderInstance.md:40]:
```suggestion
PS C:\> New-AzSapMonitorProviderInstance -ResourceGroupName nancyc-hn1 -Name ps-sapmonitorins-t01 -SapMonitorName yemingmonitor -ProviderType SapHana -HanaHostname 'hdb1-0' -HanaDatabaseName 'SYSTEMDB' -HanaDatabaseSqlPort 30015 -HanaDatabaseUsername SYSTEM -HanaDatabasePassword (ConvertTo-SecureString "Manager1" -AsPlainText -Force)
```

File: src/HanaOnAzure/examples/New-AzSapMonitorProviderInstance.md [hunk 2]
```diff
@@ -1,18 +1,22 @@
-### Example 1: {{ Add title here }}
+### Example 1: Create an instance of SAP monitor by string for HANA
 ```powershell
-PS C:\> {{ Add code here }}
+PS C:\> New-AzSapMonitorProviderInstance -ResourceGroupName nancyc-hn1 -Name ps-sapmonitorins-t01 -SapMonitorName yemingmonitor -Type SapHana -HanaHostname 'hdb1-0' -HanaDatabaseName 'SYSTEMDB' -HanaDatabaseSqlPort 30015 -HanaDatabaseUsername SYSTEM -HanaDatabasePassword (ConvertTo-SecureString "Manager1" -AsPlainText -Force)
```
Comment (isra-fel) [inline src/HanaOnAzure/examples/New-AzSapMonitorProviderInstance.md]:
```suggestion
PS C:\> New-AzSapMonitorProviderInstance -ResourceGroupName nancyc-hn1 -Name ps-sapmonitorins-t01 -SapMonitorName yemingmonitor -ProviderType SapHana -HanaHostname 'hdb1-0' -HanaDatabaseName 'SYSTEMDB' -HanaDatabaseSqlPort 30015 -HanaDatabaseUsername SYSTEM -HanaDatabasePassword (ConvertTo-SecureString "Manager1" -AsPlainText -Force)
```

File: src/HanaOnAzure/examples/New-AzSapMonitorProviderInstance.md [hunk 3]
```diff
@@ -1,18 +1,22 @@
...
 ```
 
-{{ Add description here }}
+This command creates an instance of SAP monitor by string for HANA.
 
-### Example 2: {{ Add title here }}
+### Example 2: Create an instance of SAP monitor by key vault for HANA
 ```powershell
-PS C:\> {{ Add code here }}
+PS C:\> New-AzSapMonitorProviderInstance -ResourceGroupName nancyc-hn1 -SapMonitorName sapMonitor-vayh7q-test -Type SapHana -HanaHostname 'hdb1-0' -HanaDatabaseName 'SYSTEMDB' -HanaDatabaseSqlPort 30015 -HanaDatabaseUsername SYSTEM -HanaDatabasePasswordSecretId https://kv-9gosjc-test.vault.azure.net/secrets/hanaPassword/bf516d1dfcc144138e5cf55114f3344b -HanaDatabasePasswordKeyVaultResourceId /subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f/resourceGroups/costmanagement-rg-8p50xe/providers/Microsoft.KeyVault/vaults/kv-9gosjc-test -Name sapins-kv-test
```
Comment (isra-fel) [inline src/HanaOnAzure/examples/New-AzSapMonitorProviderInstance.md]:
```suggestion
PS C:\> New-AzSapMonitorProviderInstance -ResourceGroupName nancyc-hn1 -SapMonitorName sapMonitor-vayh7q-test -ProviderType SapHana -HanaHostname 'hdb1-0' -HanaDatabaseName 'SYSTEMDB' -HanaDatabaseSqlPort 30015 -HanaDatabaseUsername SYSTEM -HanaDatabasePasswordSecretId https://kv-9gosjc-test.vault.azure.net/secrets/hanaPassword/bf516d1dfcc144138e5cf55114f3344b -HanaDatabasePasswordKeyVaultResourceId /subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f/resourceGroups/costmanagement-rg-8p50xe/providers/Microsoft.KeyVault/vaults/kv-9gosjc-test -Name sapins-kv-test
```

File: src/HanaOnAzure/examples/New-AzSapMonitorProviderInstance.md (hunk 2 above)
Comment (LucasYao93) [reply src/HanaOnAzure/examples/New-AzSapMonitorProviderInstance.md]: Done, Thanks

File: src/HanaOnAzure/examples/New-AzSapMonitorProviderInstance.md (hunk 3 above)
Comment (LucasYao93) [reply src/HanaOnAzure/examples/New-AzSapMonitorProviderInstance.md]: Done, Thanks

**Pull request merged**